apps/server/src/shared/domain/types/card.types.ts

```diff
@@ -13,7 +13,7 @@ export type ICardCProps = {
 	cardType: CardType;
 	creator: User;
 	draggable: boolean;
-	visibleAtDate: Date;
+	visibleAtDate?: Date;
```

Comment:
why is visibleAtDate reflected here instead of in `ITaskCardProps`?
the difference of type `ITaskCardProps` and interface `ITaskCard` in task-card entity is confusing